### ISSUE TITLE
Douse field Psynergy is different from GBA games

### ISSUE THREAD

**Issue by MilesBHuff** [jjppof/goldensun_html5#426]
The Douse Psynergy, when used in the field in the GBA games, has a short delay before the last raindrop comes out;  but the animation in the GSHTML5 engine lacks this nuance.
https://youtu.be/4eGLWZ5QgJ8?t=76

**Comment by MilesBHuff** [comment]
GSHTML5 is also missing the sound-effects, but you probably know that one already.

**Comment by jjppof** [comment]
> but the animation in the GSHTML5 engine lacks this nuance.

Yeah, I avoided it at the time I implemented douse... I found unnecessary one extra code section for a single drop. Will keep this one open, in the future I may add this drop.

> GSHTML5 is also missing the sound-effects, but you probably know that one already.

yes!
